extensions/amp-access/0.1/jwt.js

```diff
@@ -80,9 +136,27 @@ export class JwtHelper {
       header: tryParseJson(decodeBase64WebSafe(parts[0]), invalidToken),
       payload: tryParseJson(decodeBase64WebSafe(parts[1]), invalidToken),
       verifiable: `${parts[0]}.${parts[1]}`,
-      sig: decodeBase64WebSafe(parts[2]),
```

Comment:
No. I mean `getMode().development` - i.e. publisher dev mode, not ours :) We still want it to be on the PROD binary, just only activate when publishers debug their pages.